docs/development-guidelines/development-process.md

```diff
@@ -0,0 +1,38 @@
+## Development Guidelines - Process
+
+For cross-functional teams working on building a template, the following guidelines are recommended.
+
+- Create labels that reflect the different dimensions for features and operations. For example:
+
+| Label Name | Description                          |
+| ---------- | ------------------------------------ |
+| AI         | AI implementation and implementation |
+| AZD . Bicep| AZD config and Bicep related         |
```

Comment:
What about having 2 separate labels for `AZD` and `Infra` issues?
`Infra` might also be preferable than `Bicep` here as I believe there's also Terraform support for infra.

Reply:
Yes, we should do that

Reply:
done